cli/azd/pkg/environment/data_store.go

```diff
@@ -0,0 +1,38 @@
+package environment
+
+import (
+	"context"
+
+	"github.com/azure/azure-dev/cli/azd/pkg/contracts"
+)
+
+// DataStore is the interface for the interacting with the persistent storage of environments.
+
+type RemoteKind string
+
+const (
+	RemoteKindAzureStorage RemoteKind = "AzureStorage"
+)
+
+type DataStore interface {
+	// Gets the path to the environment .env file
```

Comment:
Should this intertace be exposed publicly? If so, we should maybe consider explaining what a `path` is.

Reply:
This has been renamed to `EnvPath` to be more clear.